internal/podman/client.go

```diff
@@ -140,35 +142,34 @@ type client struct {
 
 func (o *client) Diagnostics(ctx context.Context) *Diagnostic {
 	d := &Diagnostic{
-		Installed: true,
+		Installed:    true,
+		MachineFound: true,
 	}
 
 	err := Installed()
 	if err != nil {
 		d.Installed = false
-		return d
+		d.Errors = append(d.Errors, fmt.Errorf("failed to detect podman installed: %w", err).Error())
 	}
 
-	d.Version, err = o.Version(ctx)
+	d.Version, _ = o.Version(ctx)
```

Comment:
Should we still be catching this error? I see it's handled below

Reply:
oops, yes, thanks